tests/platform_tests/daemon/test_ledd.py

```diff
@@ -145,26 +113,27 @@ def test_pmon_ledd_term_and_start_status(check_daemon_status, duthosts, rand_one
     @summary: This test case is to check the ledd terminated and restarted status
     """
     duthost = duthosts[rand_one_dut_hostname]
+
+    if "201811" in duthost.os_version or "201911" in duthost.os_version:
```

Comment:
@sujinmkang what is the difference in behavior for SIGTERM across different version?

Reply:
@prgeor master and 202012 daemon has the signal_handler implemented to return the exitcode (128 + signal_number) but previous version just return exit code 0 for sigterm by systemd.

Reply:
Do you really want to skip this test for 201811 and 201911? If not, it would be better we handle them differently here or make the exitcode same across 201811, 201911. What about 202012?

Reply:
@prgeor this change was based on the discussion during 202012 issue triage. If there would be any back port happening, then we will need another changes for this.